lib/Runtime/Library/InJavascript/Intl.js

```diff
@@ -125,6 +125,7 @@
         slice(array, start, end) { return callInstanceFunc(platform.builtInJavascriptArrayEntrySlice, array, start, end); },
         concat(array, ...els) { return callInstanceFunc(platform.builtInJavascriptArrayEntryConcat, array, ...els); },
         filter(array, func) { return callInstanceFunc(platform.builtInJavascriptArrayEntryFilter, array, func); },
+        unique(array) { return _.filter(array, (v, i) => _.arrayIndexOf(array, v) === i); },
```

Comment:
>unique [](start = 8, length = 6)

wondering whether this implementation is the most efficient way to do this -- consult lodash?

Reply:
It likely isnt, but my understanding was that our Set is bad too, and those were the only two idiomatic implementations i could think of.

Reply:
FWIW, a quick consult of lodash suggests a basic loop with a `seen` array for small src arrays and Array.from(new Set(src)) if src is large.